src/uct/sm/self/self_ep.c

```diff
@@ -136,3 +136,11 @@ ssize_t uct_self_ep_am_bcopy(uct_ep_h tl_ep, uint8_t id,
 
     return length;
 }
+
+ucs_status_t uct_self_ep_check(uct_ep_h ep, unsigned flags, uct_completion_t *comp)
+{
+    comp->count++;
+    comp->func(comp, UCS_OK);
```

Comment:
no, this is same semantic as for zcopy/flush:
UCS_OK - completed, callback not called
INPROGRESS - started, callback will be called
ERR_NO_RESOURCE - not started, user should retry
ERR_xx - other error